core/src/main/java/org/mule/runtime/core/internal/policy/DefaultSourcePolicyProcessorFactory.java

```diff
@@ -17,19 +16,13 @@
  */
 public class DefaultSourcePolicyProcessorFactory implements SourcePolicyProcessorFactory {
 
-  private final PolicyStateHandler policyStateHandler;
-
   /**
    * Creates a new instance of the default factory for {@link SourcePolicy}s.
-   *
-   * @param policyStateHandler the state handler for policies.
    */
-  public DefaultSourcePolicyProcessorFactory(PolicyStateHandler policyStateHandler) {
-    this.policyStateHandler = policyStateHandler;
-  }
+  public DefaultSourcePolicyProcessorFactory() {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1186))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7629&issues=AWiGlnXlotIXnmsUonia&open=AWiGlnXlotIXnmsUonia)

[ISSUE_KEY]: AWiGlnXlotIXnmsUonia